bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-import.component.html

```diff
@@ -1,13 +1,13 @@
 <sm-header></sm-header>
 
-<form #form (ngSubmit)="submit()" class="tw-max-w-xl">
+<form [formGroup]="formGroup" [bitSubmit]="submit" class="tw-max-w-xl">
   <div>
     <label>{{ "fileUpload" | i18n }}</label>
     <div class="file-selector">
       <button type="button" class="btn btn-outline-primary" (click)="fileSelector.click()">
         {{ "chooseFile" | i18n }}
       </button>
-      {{ this.fileSelected ? this.fileSelected.name : ("noFileChosen" | i18n) }}
+      {{ this.formGroup.get("fileSelected")?.value?.name ?? ("noFileChosen" | i18n) }}
```

Comment:
This should work.

```suggestion
      {{ formGroup.value.fileSelected?.name ?? ("noFileChosen" | i18n) }}
```

You could also set `accept="application/JSON"` on the file selector which by default filters out non json files.

Reply:
Updated, great point, thank you!